mycroft/skills/audioservice.py

```diff
@@ -121,6 +121,23 @@ def resume(self):
         """ Resume paused playback. """
         self.bus.emit(Message('mycroft.audio.service.resume'))
 
+    def seek(self, seconds=1):
+        """ seek X seconds """
```

Comment:
Add the *Arguments:* section to these docstrings

Reply:
:+1: forgot about that